man/systemd.exec.xml

```diff
@@ -1439,10 +1439,10 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         <constant>SIGSYS</constant> signal (whitelisting). If the first character of the list is <literal>~</literal>,
         the effect is inverted: only the listed system calls will result in immediate process termination
         (blacklisting). If the setting is prefixed with <literal>~</literal>, each system call or set of system call
-        optionally takes an <literal>errno</literal> error number name such as <constant>EPERM</constant>,
-        <constant>EACCES</constant> or <constant>EUCLEAN</constant> to return when the system call is triggered,
-        instead of terminating the process immediately. The pair of a system call and errno name needs to be
-        separated by colon. The errno name specified in the setting takes precedence of one given by
+        optionally takes <constant>0</constant> or an <literal>errno</literal> error number name such as
+        <constant>EPERM</constant>, <constant>EACCES</constant> or <constant>EUCLEAN</constant> to return when the
+        system call is triggered, instead of terminating the process immediately. The pair of a system call and errno
+        name needs to be separated by colon. The errno name specified in the setting takes precedence of one given by
```

Comment:
Hmm, why just allow any number (between 0 and 255)? This way people can use errno values which don't have declared names. The documentation would also be simpler, because you'd just say "an errno error name or number (between 0 and 255)".

Reply:
OK.
In the man page [seccomp_rule_add(3)](http://man7.org/linux/man-pages/man3/seccomp_rule_add.3.html), `SCMP_ACT_ERRNO()` takes `uint16_t`. So, should we support from 0 to 65535? Or, we limit the value from 0 to 255?

Reply:
0 to 255. exit(3) says that the return value is ORed with 255.

Reply:
OK. Thanks.

Reply:
Updated.